snap/info.go

```diff
@@ -743,9 +744,46 @@ type AppInfo struct {
 
 // ScreenshotInfo provides information about a screenshot.
 type ScreenshotInfo struct {
-	URL    string
-	Width  int64
-	Height int64
+	URL    string `json:"url"`
+	Width  int64  `json:"width,omitempty"`
+	Height int64  `json:"height,omitempty"`
+}
+
+type MediaInfo struct {
+	Type   string `json:"type"`
+	URL    string `json:"url"`
+	Width  int64  `json:"width,omitempty"`
+	Height int64  `json:"height,omitempty"`
+}
+
+func (mi MediaInfo) Screenshot() ScreenshotInfo {
```

Comment:
this, as is, is strange given that it doesn't check type

Reply:
maybe i should make it private (it's just a helper for `Screenshots()`)